docs/resources/sagemaker_image_version.md

```diff
@@ -10,7 +9,45 @@ description: |-
 
 Resource Type definition for AWS::SageMaker::ImageVersion
 
+## Example Usage
 
+To create a SageMaker Image version resource.
+```terraform
+locals {
+  tags = [
+    {
+      key   = "Name"
+      value = "Sample IAM Role"
+    },
+    {
+      key   = "Environment"
+      value = "Development"
+    },
+    {
+      key   = "Modified By"
+      value = "AWSCC"
+    }
+  ]
+}
+
+variable "image_name" {
+  type = string
+}
+
+resource "awscc_sagemaker_image" "example" {
+  image_name     = var.image_name
+  image_role_arn = awscc_iam_role.example.arn
+  tags           = local.tags
+}
+
+resource "awscc_sagemaker_image_version" "test" {
+  image_name = var.image_name
+  base_image = "012345678912.dkr.ecr.us-west-2.amazonaws.com/image:latest"
```

Comment:
- Should the base_image be tied to the `awscc_sagemaker_image` ?
- Would recommend moving the `awscc_sagemaker_image_version` example config to the top.
- The tags should be updated as there are no IAM roles here. You can just use below one to keep it consistent across examples.
```
    {
      key   = "Modified By"
      value = "AWSCC"
    }
```

Reply:
@quixoticmonk  base_image cannot be tied with `awscc_sagemaker_image` since it expects image name but the output of `awscc_sagemaker_image` is only ARN. So we have to use  `depends_on`.

I have updated the rest other changes.

Reply:
`awscc_sagemaker_image` has `image_name` in the definition itself.